app.py

```diff
@@ -144,14 +143,12 @@ def auth_callback():
         user.email = email
         user.store = store
         user.admin = True
-
     db.session.add(user)
     db.session.commit()
 
     # Log user in and redirect to app home
     flask.session['userid'] = user.id
-    return flask.redirect(flask.url_for('index'))
-
+    return flask.redirect(app.config['APP_URL'])
```

Comment:
This is the bug we fixed together, as the route for index doesn't exists. This might not be the best fix, but it fixes it :)